### ISSUE TITLE
Scene Settings: Updating fbx with Level of Detail modifier fails in Asset Processor

### ISSUE THREAD

**Issue by LB-MichalRodzos** [o3de/o3de#2390]
**Describe the bug**
It has been observed, that updating fbx with Level of Detail modifier fails in Asset Processor. This issue occurs only when some entries are added to the modifier.
Please refer to the attached video/screenshot for more information.

**To Reproduce**
1. Find _test.fbx_ in the Asset Browser.
2. Open it in the _Scene Settings_.
3. Add _Level of Detail_ modifier.
4. Add at least one entry to the LOD modifier.
5. Press the Update button.

**Expected behavior**
Fbx can be updated with Level of Detail modifier added.

**Desktop/Device:**
- Device: PC
- OS: Windows
- Version: 10 Pro
- CPU: AMD Ryzen 5 3600 6-Core Processor
- GPU: NVIDIA GeForce RTX 2060 SUPER
- Memory: 16 GB

https://user-images.githubusercontent.com/86952318/126792210-dc9c28dc-e647-4013-9131-986b7e64ebcd.mp4

**Comment by jackalbe** [comment]
I just tested this out and it seems to be fixed.

**Comment by LB-JakubSkorupka** [comment]
I managed to reproduced this issue on recent Development build (c28de14).

https://user-images.githubusercontent.com/86951967/195810756-70d7ca7e-ecff-4e15-95f2-6fa4125531bf.mp4

**Comment by micronAMZN** [comment]
I've done some testing. The issue here seems to be that if an LOD slot (any LOD slot ) contains mesh nodes that are also in the base mesh group, it will fail to process.

This file is properly setup for LODs:

[LODTestAsset.zip](https://github.com/o3de/o3de/files/10865127/LODTestAsset.zip)

The above file sets LODs by naming convention and processes successfully.

If you add the `_lod0` mesh node (which is in the base mesh group) to any of the LOD slots [0-4], it will fail to process.

Try the following:

1. In Scene Settings, delete all the LOD slots except one.
2. Select any combination of mesh nodes for the base mesh group.
3. Select a different combination of mesh nodes for the LOD[0] slot.
4. The asset processes successfully.
5. Add mesh nodes that are selected for the base mesh group to the LOD[0] slot.
6. The asset fails to process.

**Comment by LB-MichalRodzos** [comment]
@micronAMZN  I followed steps from your comment above. I used the attached fbx file and cedar.fbx from the AutomatedTesting project. Tests were executed on Windows as well as on Linux. For every test, with the steps you provided the results were as expected. Is it enough to close the ticket as fixed? If yes, please let me know and I will close it, or if you need more information about the topic.

**Comment by micronAMZN** [comment]
@LB-MichalRodzos I don't think it should be closed. There is definitely a bug here. Though less common, there are use cases where someone might want to include the same unchanged mesh node(s) in the base mesh group and multiple LODs. My reply is just an attempt to add more information.
